docs/proposals/YYYYMMDD-template.md

```diff
@@ -24,7 +24,7 @@ superseded-by:
 <!-- BEGIN Remove before PR -->
 To get started with this template:
 1. **Make a copy of this template.**
-  Copy this template into `docs/enhacements` and name it `YYYYMMDD-my-title.md`, where `YYYYMMDD` is the date the proposal was first drafted.
+  Copy this template into `docs/enhancements` and name it `YYYYMMDD-my-title.md`, where `YYYYMMDD` is the date the proposal was first drafted.
```

Comment:
Copy this template into `docs/proposals` , actually there is no enhancements folder